dotsql.go

```diff
@@ -205,3 +208,81 @@ func Merge(dots ...*DotSql) *DotSql {
 		queries: queries,
 	}
 }
+
+// LoadFromDir reads the directory then loads all ".sql" files and uses file names (without ".sql") as query names
+func LoadFromDir(dirPath string, recuresive bool) (dotsql *DotSql, err error) {
+	files, err := ioutil.ReadDir(dirPath)
+	if err != nil {
+		return
+	}
+	queries := make(map[string]string)
+	var fileData []byte
+	for _, info := range files {
+		if info.IsDir() || filepath.Ext(info.Name()) != ".sql" {
+			continue
+		}
+		fileData, err = ioutil.ReadFile(filepath.Join(dirPath, info.Name()))
+		if err != nil {
+			return
+		}
+		queries[fileNameWithoutExtension(info.Name())] = string(fileData)
```

Comment:
how do you manage the case where files have more than one query? would name overlap in the map?

Reply:
force user to define only one query per file in directory; it prevents duplicate query names also;
so no need of scan query file for tag names.

Reply:
is that a documented behavior?

Reply:
just commented to describe the method

Reply:
just commented to describe the method

Reply:
I like the idea of loading a directory containing more than one `.sql` file, I think it's really useful. My worries come from the fact that it changes the regular behavior which is: ignoring the `--name:` tag, assuming there is one query per file, this will cause confusion to existing users that want to add a new file but keep the old one (with more than one query) and skip [`Load`](https://github.com/gchaincl/dotsql/blob/master/dotsql.go#L164) function, which calls the parser.

In my mind `LoadFromDir` should be an abstraction over [`LoadFromFile`](https://github.com/gchaincl/dotsql/blob/master/dotsql.go#L187) and [`Merge`](https://github.com/gchaincl/dotsql/blob/master/dotsql.go#L195).